src/hotspot/share/runtime/vframe.hpp

```diff
@@ -110,6 +110,7 @@ class vframe: public ResourceObj {
   // printing operations
   virtual void print_value() const;
   virtual void print();
+  void print_on(outputStream* st) const override;
```

Comment:
Unclear if this should also be virtual.

Reply:
This is virtual, taken from `ResourceObj`. The `override` indicates this, I'm basing this on: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-override

Reply:
Thanks - missed that.

Reply:
I don't know if we've added 'override' to our source base yet.